conf/config.php.j2

```diff
@@ -56,7 +56,12 @@ return array(
         // LimeSurvey developers: Set this to 2 to additionally display STRICT PHP error messages and get full access to standard templates
                 'debug'=>0,
                 'debugsql'=>0, // Set this to 1 to enanble sql logging, only active when debug = 2
-				'enableLdap'=>true
+                /* yunohost part */
+                'auth_webserver_user_map' => array( '{{ admin }}'=>'ynh_admin'),// Primary user as ynh admin
```

Comment:
I do some test : if you remove user access (1st user) from LS : then you don't have a user who can set super-admin ... 

The issue with LS is 

1. Only the 1st super-admin (userid==1) can set other super-admin
2. The 1st super-admin (userid==1) can always log by DBAuth
3. The 1st super admin can not be removed (from LS)
4. The 1st super-admin have always all the rights on all ...

Then : if you set 'ynh-user1' to super-admin at start, play a little, via yunohots : remove this admin, set other admin : the new admin don't are really a real super-admin ....

It's really the point where i didn't know how it's best to fix ....